src/pages/contact.js

```diff
@@ -48,16 +31,18 @@ function Contact({ data }) {
 					<h3 className="f2 lh-solid mb2 light-grey">Contact us</h3>
 				</div>
 				<div className="grid-item w-10 push-1 w-7-m push-0-m pv5-m">
-					<ContactCard
-						className="flex-column f3 lh-copy dark-grey"
-						addressClassName="mb2-m"
-						contactClassName="hover-light-grey"
-						tel={tel}
-						email={email}
-						street={street}
-						city={city}
-						zip={zip}
-					/>
+					{offices.map(office => (
```

Comment:
If we deconstruct here, we can tidy it up a little 

```suggestion
					{offices.map(({ name, tel, address: { street, city, zip }}) => (
```

Reply:
I find that suggestion harder to read than the original, and it's not a pattern we use in other places of the code base:

<img width="424" alt="screenshot 2019-01-17 at 9 47 13 am" src="https://user-images.githubusercontent.com/1170905/51309702-e7c02500-1a3c-11e9-92fc-e30eac921829.png">

So going to not implement the change unless you and others have strong opinions on it

Reply:
I suppose my main reason for deconstructing is only passing through what you intend to consume. Since everything in the object is being used, it makes sense to pass it through untouched.